src/main/java/seedu/duke/commands/EditCommand.java

```diff
@@ -18,7 +18,7 @@
     private static final String EDIT_TASK_SUCCESS = StringConstants.EDIT_TASK_SUCCESS;
     private static final String EDIT_TASK_WITH_MODULE_SUCCESS = StringConstants.EDIT_TASK_WITH_MODULE_SUCCESS;
     private static final String TASK_DESCRIPTION = StringConstants.TASK_DESCRIPTION_STR;
-    private static final String ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
+    private static final String TASK_ESTIMATED_WORKING_TIME = StringConstants.TASK_ESTIMATED_WORKING_TIME_STR;
```

Comment:
If both are private static final,  should we still declare it repeatedly? How about just use the constant directly.